test/kv679_eqc.erl

```diff
@@ -0,0 +1,493 @@
+%% -------------------------------------------------------------------
+%%
+%% eqc model/tests for kv679 type bugs
+%%
+%% Copyright (c) 2017 bet365, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+-module(kv679_eqc).
+
+-ifdef(EQC).
+-include_lib("eqc/include/eqc.hrl").
+-include_lib("eqc/include/eqc_statem.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+-compile(export_all).
+
+-define(P1, p1).
+-define(REPLICAS, [p2, p3, f1, f2, f3]).
+
+-record(state, {
+          pfhead=?P1,
+          %% Actor Ids for replicas in the system
+          replicas= ?REPLICAS,
```

Comment:
Seems like this is never modified, right? Would be slightly simpler to just use the `?REPLICAS` constant directly in the code and kill this state field. Though if there are future plans to make this field mutable then it might be worth keeping it around.